README.md

```diff
@@ -21,7 +21,7 @@ The token itself is largely based on COMP and Ampleforth which have undergone au
 
 The rebaser may also have bugs - but has been tested in multiple scenarios. It is restricted to Externally Owned Accounts (EOAs) calling the rebase function for added security. SafeMath is used everywhere.
 
-If you feel uncomfortable with these disclosures, don't stake or hold HAM. If the community votes to fund an audit, or the community is gifted an audit, there is no assumption that the original devs will be around to implement fixes, and is entirely at their discretion.
+If you feel uncomfortable with these disclousures, don't stake or hold HAM. If the community votes to fund an audit, or the community is gifted an audit, there is no assumption that the original devs will be around to implement fixes, and is entirely at their discretion.
```

Comment:
"Disclosures" is actually correct https://en.wikipedia.org/wiki/Disclosure

Reply:
the code has obviously changed since I first cloned the project, I did not make this change. My apologies.